core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -521,20 +521,20 @@ class GroupMetadataManager(brokerId: Int,
     val topicPartition = new TopicPartition(Topic.GROUP_METADATA_TOPIC_NAME, offsetsPartition)
     if (addLoadingPartition(offsetsPartition)) {
       info(s"Scheduling loading of offsets and group metadata from $topicPartition")
-      scheduler.schedule(topicPartition.toString, () => loadGroupsAndOffsets(topicPartition, onGroupLoaded))
+      scheduler.schedule(topicPartition.toString, () => loadGroupsAndOffsets(topicPartition, onGroupLoaded, time.milliseconds()))
```

Comment:
Hmm.. Does this do what's intended? Wouldn't `time.milliseconds` only be evaluated when the callback is invoked?

Reply:
True, has to be evaluated before passing in. Done.